qiita_ware/test/test_search.py

```diff
@@ -0,0 +1,53 @@
+from unittest import TestCase, main
+
+from qiita_core.util import qiita_test_checker
+from qiita_db.search import QiitaStudySearch
+from qiita_db.user import User
+from qiita_ware.search import count_metadata, filter_by_processed_data
+
+
+@qiita_test_checker()
+class SearchTest(TestCase):
+    """Tests that the search helpers work as expected"""
+    def test_count_metadata(self):
+        search = QiitaStudySearch()
+        results, meta_cols = search('study_id = 1 AND ph > 0',
```

Comment:
It would be much more useful if the inputs to `count_metadata` where explicitly described instead of fetching from the database. It is difficult to evaluate this test without knowing the inputs.